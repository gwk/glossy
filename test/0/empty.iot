{
  'args': ['-test',
    'a b c',
    'abc def',
  ],
}
