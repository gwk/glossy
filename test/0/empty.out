
arg1: 'a b c'
arg1:1:1-6: invalid
a b c
~~~~~

arg2: 'abc def'
arg2:1:1-8: invalid
abc def
~~~~~~~
